forth_src/test.fs

```diff
@@ -18,7 +18,7 @@ parse-name mmldemo included
 1 fakekeys \ exits demo
 parse-name spritedemo included
 
-latest ! to here
+to latest to here
```

Comment:
I am not sure why this is not using `MARKER`

Reply:
Fixing this uncovered a bug in `MARKER`.  Good test harness :)